### ISSUE TITLE
Get some of that sweet, sweet Mozilla cash

### ISSUE THREAD

**Issue by jacobian** [pybee/paying-the-piper#43]
Mozilla's [launching an Open Source Support Program](https://blog.lizardwrangler.com/2015/10/23/mozilla-launches-open-source-support-program/) to "a) a “give back” [to] Open Source and Free Software projects that Mozilla relies on; and b) “give forward” [to] other projects where financial resources from Mozilla can make our entire community more successful." 

It's unclear exactly how to engage or ask for money, but [the wiki has some more information](https://wiki.mozilla.org/MOSS) along with a [list of (known) projects Mozilla relies on](https://wiki.mozilla.org/MOSS/Projects_in_use_by_Mozilla).

**Comment by nanuxbe** [comment]
+1
Django is even listed twice on the projects page, double the chance to get funding?

**Comment by freakboy3742** [comment]
While Mozilla's proposal is fantastic, and certainly welcome - is there anything we can do to encourage other organisations to do the same thing? What can we do to encourage this sort of patronage at a larger scale?

**Comment by jacobian** [comment]
Yeah that's a good question, isn't it.

One thing I know is spending the money -- **effectively**. When I've talked to people who fundraise for a living, one thing they mention time and again is that the best way to get repeated donors is for them to know how their money was spent, and feel good about it. IOW, **demonstrate results**. So at least here in the Mozilla case, we can (in theory) help by spending their money really well :)

More broadly I think the model here of a central foundation giving out grants is a very good one, and one I'd like to see extended (see also #36). A central foundation can afford fundraisers, spearhead major drives, set standards around accountability and results, etc. Economies of scale can, I think, work for open source too. I could see the PSF, Node Foundation, etc doing more of this work, and if Mozilla proves the model maybe that'll help.

**Comment by freakboy3742** [comment]
@jacobian Agreed - appearing to be successful probably more important than actually _being_ successful.

A magic keyword to throw in the mix here is [Corporate Social Responsibility](https://en.wikipedia.org/wiki/Corporate_social_responsibility). A lot of larger organisations have a CSR policy (I was in a meeting yesterday with one), and being able to say that a specific initiative addresses some aspect of CSR is one way to lubricate the funding wheels.

Exactly which aspects of CSR any given organization cares about will obviously be dependent on the organization (and in some cases, local conditions - in Australia, supporting indigenous and remote communities is a big CSR trigger point), but if we can push those buttons as part of a fundraising push, it could work to our collective advantage.

**Comment by tleeuwenburg** [comment]
I would happily put $100 per year towards membership of something that I though was going to make a difference in this area, and chalk it up as a professional membership expense. It would be an easy decision.

**Comment by freakboy3742** [comment]
@tleeuwenburg "Professional Membership" is an interesting angle... do you have any idea what the ATO's minimum requirement for a professional society is? Could the DSF just start selling "professional memberships" without actually doing anything extra like providing formal professional training?

**Comment by cjrh** [comment]
"Professional membership" is a great idea.  I have to pay something like $380 a year to belong to a chemical engineers association thing, to give you an idea of what other orgs charge.

**Comment by freakboy3742** [comment]
@cjrh Same question for you, then: What does your professional organization have to actually "do" in order to qualify for the tax deduction? Have they got a lock on professional practice? (i.e., you can't get hired unless you're a member)? Or is a voluntary organization that is looked on favourably by employers? Do they have to provide some sort of service, or accreditation, or ongoing training?

**Comment by cjrh** [comment]
The org in question is [IChemE](http://www.icheme.org/)

They do not (yet) have a lock on professional practice, but they're trying, unsurprisingly, to move in that direction.  Specifically in Queensland, Australia, there is a lock on being a "certified engineer", but you can do either [RPEQ](http://rpeng.org.au/rpeq/) which is a Queensland-only thing, or IChemE.  Certification is normally not required in most countries, but in Queensland it is required for several formal activities, one of which is to have "sign-off" responsibility on design work.

Internationally, certification is voluntary, but it is becoming voluntary in the same way that LinkedIn is voluntary, i.e., you _can_ choose not to, but it may be better for your career if you do.

I have no idea what they do around tax deductions or any of those organizational things.  My impression is that it is looked on favourably by employers. In Queensland specifically, it's pretty hard to even get a job without one, the current economic climate in oil & gas notwithstanding.

They require ongoing training from your side in order to "retain your certification", and they organize ongoing webinars and things that you can use to contribute hours to your annual quota.  When I used to be employed, I know my company paid for my professional membership and I think they got some kind of deduction out of that.  The invoice disappeared into the black corporate void and I lost track of it.

**Comment by tleeuwenburg** [comment]
@freakboy3742  I haven't got the real answer. However, there are many professional associations in the world -- I would suggest finding one and asking them what they did! :) You may find the answers online at the ATO. I'll see if I can find out through some Googling at some point.

**Comment by tleeuwenburg** [comment]
I think you need to form a non-profit organisation, which requires the creation of the legal entity and management of it, then "the rest is just implementation". Here in Australia, You might find that you could do this within the auspices of LCA and a body of the parent organisation. If you are looking globally, a lot of organisations seem to incorporate in the US, I think because of how they recognise that on tax returns. I'd get in touch with someone who understand the relevant legal frameworks.

My advice would be to form a clear idea of what the body would do, what its policies would be, then take that to someone who know how to get to the next level.

I'd be cautious about pushing the 'certification' angle too far though -- have a chat to Steve Holden about that :)

**Comment by nayafia** [comment]
So I just saw on the MOSS mailing list that they only received 12 applications by the round 1 deadline. Total amount requested was around $900k.

I'm shocked the count is this low, given how many applications Stripe got for its grant program (130 for 2 slots). My two guesses are either: 1) not enough devs were aware of the program, or 2) Stripe gave individual grants whereas Mozilla seems to be targeting larger organizations, and it's a lot harder to mobilize the latter.

Hoping future details will help paint a picture of the applicants, but did anyone here apply? Why/why not?

**Comment by lovelydinosaur** [comment]
> Hoping future details will help paint a picture of the applicants

Indeed. They're planning on announcements during a big get-together next week - so will be clearer then.

One reason for applications being lower than Stripe will be that the criteria included that the project must be currently in use at Mozilla. Stripe's grant program doesn't appear to have had a similar constraint.

**Comment by nayafia** [comment]
ah, makes sense. I had seen that MOSS was intended to support both projects that benefit Mozilla and the ecosystem as a whole, but just noticed that currently they are only supporting the former.

**Comment by freakboy3742** [comment]
I think part of the problem is that the rules around what they were going to fund, and how you applied, were unclear, especially at time of launch of the program. The DSF was obviously interested in the program, but the application process wasn't at all clear.

Stripe, on the other hand, had a clearly identifiable application process and set of requirements for applicants.

**Comment by ericholscher** [comment]
Agreed. I almost missed the application deadline because it was super unclear, and wasn't included in the initial announcement or wiki page (You had to dig in the mailing list to find it). I didn't see any mention of the deadline as it approached in my social media either, though the initial announcement got a lot of play.
